### PR TITLE
changed system from hl7 terminology to fhir nhs

### PR DESCRIPTION
**Pull request by conner-jowitt**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: models/examples/example-2-acute-nominated-pharmacy/PrepareRequest-FhirMessageUnsigned.json [hunk 1]
```diff
@@ -77,7 +77,7 @@
         "courseOfTherapyType": {
           "coding": [
             {
-              "system": "https://fhir.nhs.uk/R4/CodeSystem/UKCore-PrescriptionType",
+              "system": "https://fhir.nhs.uk/R4/CodeSystem/medicationrequest-course-of-therapy",
```
Comment (mfjarvis) [inline models/examples/example-2-acute-nominated-pharmacy/PrepareRequest-FhirMessageUnsigned.json:80]: I think the system is http://terminology.hl7.org/CodeSystem/medicationrequest-course-of-therapy for this case

File: models/examples/example-1-repeat-dispensing/PrepareRequest-FhirMessageUnsigned.json [hunk 2]
```diff
@@ -74,7 +74,7 @@
         "courseOfTherapyType": {
           "coding": [
             {
-              "system": "https://fhir.nhs.uk/R4/CodeSystem/UKCore-PrescriptionType",
+              "system": "https://fhir.nhs.uk/R4/CodeSystem/medicationrequest-course-of-therapy",
               "code": "repeat-dispensing",
```
[the remark this answers is not in the document]
Comment (mfjarvis) [reply models/examples/example-1-repeat-dispensing/PrepareRequest-FhirMessageUnsigned.json:78]: convertCourseOfTherapyTypeCode will also need changing to expect the new values:
- acute
- continuous
- continuous-repeat-dispensing

File: models/schemas/Coding-CourseOfTherapyType.yaml [hunk 3]
```diff
@@ -8,8 +8,8 @@ properties:
     type: string
     description: The identification of the code system that defines the meaning of the symbol in the code.
     enum: 
-      - "https://fhir.nhs.uk/R4/CodeSystem/UKCore-PrescriptionType"
-    example: "https://fhir.nhs.uk/R4/CodeSystem/UKCore-PrescriptionType"
+      - "https://fhir.nhs.uk/R4/CodeSystem/medicationrequest-course-of-therapy"
```
Comment (mfjarvis) [inline models/schemas/Coding-CourseOfTherapyType.yaml:11]: The enum will need to allow for both code systems:
- http://terminology.hl7.org/CodeSystem/medicationrequest-course-of-therapy
- https://fhir.nhs.uk/R4/CodeSystem/medicationrequest-course-of-therapy

File: coordinator/tests/services/spine-communication.spec.ts [hunk 4]
```diff
@@ -1,5 +1,5 @@
 import 'jest'
-import moxios from 'moxios'
```
Comment (conner-jowitt) [inline coordinator/tests/services/spine-communication.spec.ts:2]: This line shows as an error in intellij: 
`TS1259: Module '"/home/conner/NHSdir/electronic-prescription-service-api/coordinator/node_modules/@types/moxios/index"' can only be default-imported using the 'esModuleInterop' flag 

index.d.ts(194, 1): This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.`

File: models/schemas/Coding-CourseOfTherapyType.yaml (hunk 3 above)
Comment (conner-jowitt) [reply models/schemas/Coding-CourseOfTherapyType.yaml:11]: Added both to the spec

File: coordinator/tests/services/spine-communication.spec.ts (hunk 4 above)
Comment (ri-ch) [reply coordinator/tests/services/spine-communication.spec.ts:2]: Is anyone else seeing this problem?

**Pull request merged**